styles/prosilver/theme/rh_topictags.css

```diff
@@ -104,3 +104,11 @@ and (max-width : 480px)
 		display:	block;
 	}
 }
+
+dl.row-item {
```

Comment:
these ruels should probably be bound to rh_topictags and not change dl/dt in general